unstructured_ingest/v2/interfaces/file_data.py

```diff
@@ -43,6 +43,7 @@ class FileData(DataClassJsonMixin):
     additional_metadata: dict[str, Any] = field(default_factory=dict)
     reprocess: bool = False
     local_download_path: Optional[str] = None
+    record_display_name: Optional[str] = None
```

Comment:
Would this be more succinct? 
```suggestion
    display_name: Optional[str] = None
```